node_modules/col-core/lib/api.js

```diff
@@ -34,12 +34,9 @@ var Server = require('./server');
  * @param  {Object}         callback.err        An error object, if any
  */
 var init = module.exports.init = function(callback) {
-  // All unexpected or uncaught errors will be caught and logged here. At this point we cannot
-  // guarantee that the system is functioning properly anymore so we kill the process. When running
-  // in production, the service script will automatically respawn the instance
+  // All unexpected or uncaught errors will be caught and logged here
   process.on('uncaughtException', function(err) {
-    log.error({'err': err}, 'Uncaught exception was raised, restarting the process');
-    process.exit(1);
```

Comment:
I removed it because this would cause all of the connected websockets to be disconnected. They do reconnect after a while, but there'll be a time gap where those users won't be able to make whiteboard changes. Let me know if you would like me to undo this change.